tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -144,17 +149,19 @@ private static ArgumentParser argParser() {
 
         parser.addArgument("--num-threads")
                 .action(store())
-                .required(true)
+                .required(false)
                 .type(Integer.class)
                 .metavar("NUM-THREADS")
                 .dest("numThreads")
-                .help("The value bound of the random integers in message payload to simulate different compression ratio.");
+                .setDefault(1)
+                .help("The number of producing threads.");
 
         parser.addArgument("--value-bound")
                 .action(store())
-                .required(true)
+                .required(false)
                 .type(Integer.class)
                 .metavar("VALUE-BOUND")
+                .setDefault(0)
                 .dest("valueBound")
                 .help("The value bound of the random integers in message payload to simulate different compression ratio.");
```

Comment:
Do you have any estimation of what compression ratio can be achieved by varying the value of this config? Is this easy to get by running experiments and get the overall compression ratio at the end of each experiment?